### ISSUE TITLE
Disabled button breaks keyboard nav

### ISSUE THREAD

**Issue by jakub-g** [ariatemplates/ariatemplates#1364]
http://snippets.ariatemplates.com/samples/github.com/ariatemplates/documentation-code/samples/widgets/button/style/

The disabled button is a black hole in IE: you can't get past it via `TAB` or tab back through it using `SHIFT-TAB`
